_includes/header.html

```diff
@@ -10,7 +10,7 @@
   <div class="panel-main">
     <div class="panel-main__inner panel-inverted">
       <div class="panel-main__content">
-        <img src="/images/gear.jpg" class="user-image" alt="My Profile Photo" />
+        <img src="images/BIDS_apps_logo_black.jpg" class="user-image" alt="My Profile Photo" />
```

Comment:
is this maybe a confusion with https://github.com/BIDS-Apps/bids-apps.github.io/pull/56 ?

Reply:
you are right, I mixed up my PRs